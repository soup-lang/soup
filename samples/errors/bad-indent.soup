 func m(x, y, z)
     write(x + y)
   write(y + z)

  if false
    return

func nothing()
        return
