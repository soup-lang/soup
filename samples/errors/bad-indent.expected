samples/errors/bad-indent.soup:3
    invalid indentation
samples/errors/bad-indent.soup:5
    invalid indentation
samples/errors/bad-indent.soup:6
    invalid indentation
samples/errors/bad-indent.soup:8
    invalid indentation
